keras_nlp/models/roberta/roberta_tasks.py

```diff
@@ -55,19 +52,13 @@ class RobertaClassifier(keras.Model):
     """
 
     def __init__(
-        self,
-        backbone,
-        num_classes,
-        hidden_dim=None,
-        dropout=0.0,
-        name=None,
-        trainable=True,
+        self, backbone, num_classes, hidden_dim=None, dropout=0.0, **kwargs
```

Comment:
Not something you changed, but is `0` the correct default for dropout? If so, why are we including it?

Reply:
Yep.

https://github.com/facebookresearch/fairseq/blob/b5a039c292facba9c73f59ff34621ec131d82341/fairseq/models/roberta/model.py#L637

https://github.com/facebookresearch/fairseq/blob/b5a039c292facba9c73f59ff34621ec131d82341/fairseq/models/roberta/model.py#L515

Reply:
HF, on the other hand, has 0.1:

https://huggingface.co/roberta-base/blob/main/config.json#L9

https://github.com/huggingface/transformers/blob/bd469c40659ce76c81f69c7726759d249b4aef49/src/transformers/models/roberta/modeling_roberta.py#L1437-L1439